asset_test.go

```diff
@@ -42,9 +42,13 @@ func TestAssetGet(t *testing.T) {
 	setup()
 	defer teardown()
 
-	httpmock.RegisterResponder(
+	httpmock.RegisterResponderWithQuery(
 		"GET",
-		"https://fooshop.myshopify.com/admin/themes/1/assets.json?asset[key]=foo/bar.liquid&theme_id=1",
+		"https://fooshop.myshopify.com/admin/themes/1/assets.json",
+		map[string]string{
```

Comment:
maybe instead of this, we could have 
`params := map[string]string{"asset[key]": "foo/bar.liquid", "theme_id":1}`
then ```httpmock.RegisterResponderWithQuery(
                    "GET",
                    "https://fooshop.myshopify.com/admin/themes/1/assets.json",
                    params,
                   httpmock.NewStringResponder(200, `{"asset": {"key":"foo\/bar.liquid"}}`))```

Reply:
@andrewhoff resolved.

Reply:
ah nice